app/uk/gov/hmrc/tariffclassificationfrontend/controllers/RejectCaseController.scala

```diff
@@ -35,18 +37,31 @@ class RejectCaseController @Inject()(authenticatedAction: AuthenticatedAction,
                                      val messagesApi: MessagesApi,
                                      implicit val appConfig: AppConfig) extends RenderCaseAction {
 
+  private val form: Form[Boolean] = MandatoryBooleanForm.form("reject_case")
+
   override protected val config: AppConfig = appConfig
   override protected val caseService: CasesService = casesService
 
   override protected def redirect: String => Call = routes.CaseController.applicationDetails
   override protected def isValidCase(c: Case)(implicit request: AuthenticatedRequest[_]): Boolean = c.status == OPEN
 
   def rejectCase(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
-    getCaseAndRenderView(reference, c => successful(views.html.reject_case(c)))
+    getCaseAndRenderView(reference, c => successful(views.html.reject_case(c, form)))
```

Comment:
line 49 and line 55 can re-use a common method

Reply:
As discussed I'm going to leave this as it's already readable